sheets/3/exercise-2.tex

```diff
@@ -117,16 +117,17 @@
 
 \begin{enumerate}\setcounter{enumi}{3}
 	\item
-	Mit dem neuen Wissen wollen wir nun an unserem Code aus Teilaufgabe c) weiterarbeiten.
+	Mit diesem neuen Wissen wollen wir nun an unserem Code aus Teilaufgabe c) weiterarbeiten.
 
-	Wenn Totoro also auf einem Feld mit genau einer Nuss steht, soll er die Nuss aufsammeln, sich dann nach rechts drehen und einen weiteren Schritt machen.
+	Wenn Totoro also auf einem Feld mit genau einer Nuss steht, soll er die Nuss aufsammeln und 
+	sich dann nach rechts drehen und einen weiteren Schritt machen.
 	Wenn sich mehr als eine Nuss unter Totoro befinden, soll er die Schleife erstmals mit einem \lstinline{break} verlassen.
-	Dann soll der wieder anfangen zu laufen, bis unter ihm wieder eine oder mehrere Nüsse auftauchen.
-	
+	Dannach soll er wieder weiter laufen, bis er Nüsse findet.	
 	\item
-	Wenn sich jetzt mehr als eine Nuss unter Totoro befinden sollten, dann soll Totoro genau eine Nuss aufsammeln, sich nach links drehen und dann wieder einen Schritt nach vorne machen.
+	Wenn sich jetzt mehr als eine Nuss unter Totoro befinden, 
+	soll Totoro genau eine Nuss aufsammeln, und sich dann nach links drehen und  wieder einen Schritt nach vorne machen.
 	
-	Momentan soll Totoro noch nicht alle Nüsse aufgesammelt bekommen, ohne gegen einen Busch zu laufen.
+	Momentan soll Totoro noch nicht alle Nüsse aufgesammelen, ohne gegen einen Busch zu laufen.

```

Comment:
Instruction should indicate, that with the current code, totoro might not be able to collect all nuts without running into a bush